pyaedt/modeler/Primitives.py

```diff
@@ -231,10 +234,8 @@ def _pl_point(self, pt):
         return pt_data
 
     def _point_segment_string_array(self):
-        """Retrieve a parameter array for points and segments.
-
-        Returns the parameter array required to specify the points and segments of a polyline
-        for use in the AEDT API command ``CreatePolyline``.
+        """Retrieve the parameter arrays for specifying the points and segments of a polyline
+        for use by the :func:`pyaedt.modeler.Primitives3D.Primitives3D.create_polyline` method.
```

Comment:
```suggestion
        """Retrieve the parameter arrays for specifying the points and segments of a polyline
        used in the :class:`pyaedt.modeler.Primitives.Polyline` constructor.
```
This method is called in Primitives.py line 179, when creating the Polyline object.